content/swiftui/concepts/viewmodifier/terms/border/border.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: '.border()'
+Description: 'Applies a border to a view, which can be customized with a specified style and width.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI ViewModifiers'
+  - 'Xcode'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.border()`** view modifier in SwiftUI can be used to add a border to a view. The border can be customized with a specified style and width.
+
+## Syntax
+
+```pseudo
+View
```

Comment:
One more addition would be helpful here about what views this modifier can be used on

Reply:
Thank you for reviewing my changes @yangc95 
I have updated the documentation to reflect that the .border() view modifier can be applied to any type of View and provided a few examples. Please let me know if there are any further changes or additions you would like me to make.